_shellcomp/volt

```diff
@@ -0,0 +1,63 @@
+get_plugs() {
+	local profile=$1
+	case "$2" in
+		all) all_plugs ;;
+		this) this_plug "$profile" ;;
+		not) comm -23 <(all_plugs) <(this_plug "$profile") ;;
+		*) echo ;;
+	esac
+}
+
+all_plugs() {
+	volt list -f "{{ range .Repos }}{{ println .Path }}{{ end }}" | sed -e 's/github\.com\///' | sed -e '/^www\./d' | sort
```

Comment:
I removed those lines, because the completion is only for `username/package`.
If the packages, which have to be parsed, were repeated ( because of github and www.github duality), there would be twice the same package in the list and I didn't want this to give me any problems, so I removed it

Reply:
I see. So how about this?
`... | sed -E 's@^(www\.)?github\.com/@@' | sort -u`